dots_and_boxes/__tests__/gameLogic_test.ts

```diff
@@ -0,0 +1,74 @@
+import {IMove} from '../../common/common';
+import {aiService} from '../aiService';
+import * as gameLogic from '../gameLogic';
+import {cellColor} from '../gameLogic';
+
+describe('In Dots_and_Boxes', function () {
+  function expectMove(move: IMove<gameLogic.IState>, isIllegal: boolean): void {
+    const {
+      board: {size},
+    } = move.state;
+    if (move.state.delta) {
+      // eslint-disable-next-line no-var
+      var {direction, col, row} = move.state.delta;
```

Comment:
Instead of using -  // eslint-disable-next-line no-var,
I think you can add a small object defining this build and use it for transferring the data.